README.md

```diff
@@ -5,21 +5,79 @@
 cri-containerd is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+The current release of `cri-containerd` is `v0.1.0-alpha.1`. This release is for
+use with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Social
+This is a project within the Kubernetes SIG named `sig-node`.
+For more information about `sig-node` and the `cri-containerd` project:
+* [sig-node community site](https://github.com/kubernetes/community/tree/master/sig-node).
+* Catch us on Kubernetes (kubernetes.slack.com) in the `#sig-node` channel.
+* Mailing List: https://groups.google.com/forum/#!forum/kubernetes-sig-node
+## Getting started
+### Binary Dependencies
```

Comment:
I did provide links... but I'll also mention the new install.deps that we'll have to keep up to date :-)